src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -42,6 +42,7 @@ public static partial class Http
             public static string RevokedCertRemoteServer => GetValue("COREFX_HTTPHOST_REVOKEDCERT", "https://revoked.badssl.com/");
 
             public static string EchoClientCertificateRemoteServer => GetValue("COREFX_HTTPHOST_ECHOCLIENTCERT", "https://corefx-net-tls.azurewebsites.net/EchoClientCertificate.ashx");
+            public static string Http2ForceUnencrypted => GetValue("COREFX_HTTP2FORCEUNENCRYPTED");
```

Comment:
nit: I would prefer more whitespace in this, i.e. "COREFX_HTTP2_FORCEUNENCRYPTED"